Packs/Armis/Integrations/Armis/Armis.yml

```diff
@@ -0,0 +1,491 @@
+category: Utilities
+commonfields:
+  id: Armis
+  version: -1
+configuration:
+- display: Server URL
+  name: url
+  required: true
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- defaultvalue: '10'
+  display: Maximum number of incidents per fetch
+  name: max_fetch
+  required: false
+  type: 0
+- defaultvalue: ACTIVE
+  display: Fetch alerts with status (UNHANDLED,SUPPRESSED,RESOLVED)
+  name: alert_status
+  options:
+  - UNHANDLED
+  - SUPPRESSED
+  - RESOLVED
+  required: false
+  type: 16
+- additionalinfo: The types of Alerts are Policy Violation, System Policy Violation,
+    Anomaly Detection
+  display: Fetch alerts with type
+  name: alert_type
+  options:
+  - Policy Violation
+  - System Policy Violation
+  - Anomaly Detection
+  required: false
+  type: 16
+- defaultvalue: Low
+  display: Minimum severity of alerts to fetch
+  name: min_severity
+  options:
+  - Low
+  - Medium
+  - High
+  required: true
+  type: 15
+- defaultvalue: 3 days
+  display: First fetch time
+  name: first_fetch
+  required: false
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Secret API Key
+  hidden: false
+  name: secret
+  required: true
+  type: 4
+- additionalinfo: use this parameter to fetch incidents using a free AQL string rather
+    than the simpler alert type, severity, etc'
+  display: Fetch Alerts AQL
+  hidden: false
+  name: free_fetch_string
+  required: false
+  type: 0
+description: Integration with Armis
```

Comment:
Please Improve the description. You can find guidelines here: 
https://xsoar.pan.dev/docs/documentation/documentation_tips#for-integrations